flow/connectors/bigquery/qvalue_convert.go

```diff
@@ -92,3 +92,13 @@ func BigQueryTypeToQValueKind(fieldType bigquery.FieldType) (qvalue.QValueKind,
 		return "", fmt.Errorf("unsupported bigquery field type: %v", fieldType)
 	}
 }
+
+func qValueKindToBigQueryTypeString(colType string) string {
+	bqType := qValueKindToBigQueryType(colType)
+	bqTypeAsString := string(bqType)
+	// string(bigquery.FloatFieldType) is "FLOAT" which is not a BigQuery type.
+	if bqType == bigquery.FloatFieldType {
+		bqTypeAsString = "FLOAT64"
+	}
+	return bqTypeAsString
```

Comment:
```suggestion
	// string(bigquery.FloatFieldType) is "FLOAT" which is not a BigQuery type.
	if bqType == bigquery.FloatFieldType {
		return "FLOAT64"
	}
	return string(bqType)
```
should update with other types from https://github.com/googleapis/google-cloud-go/blob/37ef89207989af819106ee4012498e638b8cc195/bigquery/schema.go#L337